hokusai/production.yml

```diff
@@ -52,10 +60,10 @@ spec:
               containerPort: 8080
           resources:
             requests:
-              cpu: 200m
-              memory: 256Mi
-            limits:
+              cpu: 100m
```

Comment:
most of the time, [cpu util ](https://app.datadoghq.com/dashboard/2n5-6tr-ypp/kubernetes-cpumemory-usage-by-deployment?from_ts=1616080309689&live=true&tile_size=s&to_ts=1616685109689&tpl_var_deployment=convection-web&tpl_var_env=production&tpl_var_hpa=convection-web)is near zero. and hpa has not scaled up at all.